### ISSUE TITLE
Aria2 cannot be downloaded, but others such as deluge and qbittorrent can

### ISSUE THREAD

**Issue by moshengrenzenme** [aria2/aria2#1946]
I made a torrent file myself and shared it with my friends. He can't download it with aria2, but others such as deluge and qbittorrent can find my aria2, but they can't find themselves

**Comment by alpha7504** [comment]
I'm facing the same issue, I can't download any torrent file using aria2, normal http/s based downloads are fine but torrent download either by magnet or torrent file itself is not working. I can download the same torrent file using utorrent or bit torrent just fine, so something is going on with aria2.

**Comment by jazz1611** [comment]
do you try look this bug? https://github.com/aria2/aria2/issues/1903

**Comment by moshengrenzenme** [comment]
> do you try look this bug? #1903

 Sorry, I think they are different. I am the first peer (I don't know whether the translation is correct). My friend can find my aria2c and download it normally using qbittorrent, but he can't find my aria2c using aria2c

**Comment by jazz1611** [comment]
> > do you try look this bug? #1903
> 
> Sorry, I think they are different. I am the first peer (I don't know whether the translation is correct). My friend can find my aria2c and download it normally using qbittorrent, but he can't find my aria2c using aria2c

are you sure the torrent file is not hybird torrent? please make sure it have same "Info Hash v1"

**Comment by alpha7504** [comment]
> do you try look this bug? #1903
Yes, I looked now, and it seems this is the issue I'm facing... Weird thing is, it used to work fine probably around 2 months back. Why torrents stopped working all off a sudden?

**Comment by jazz1611** [comment]
> Yes, I looked now, and it seems this is the issue I'm facing... Weird thing is, it used to work fine probably around 2 months back. Why torrents stopped working all off a sudden?

if the user creates a new torrent without paying attention, it will create a hybrid torrent format then try create Torrent have only Info Hash v1 and you can try again with aria2c
![image](https://user-images.githubusercontent.com/1846222/178305993-3ea188f6-62c7-41dd-ad0f-2cf35890ddc0.png)

**Comment by moshengrenzenme** [comment]
> > Yes, I looked now, and it seems this is the issue I'm facing... Weird thing is, it used to work fine probably around 2 months back. Why torrents stopped working all off a sudden?
> 
> if the user creates a new torrent without paying attention, it will create a hybrid torrent format then try create Torrent have only Info Hash v1 and you can try again with aria2c ![image](https://user-images.githubusercontent.com/1846222/178305993-3ea188f6-62c7-41dd-ad0f-2cf35890ddc0.png)

 Thank you. I'll try
